config/settings.yml

```
@@ -35,3 +35,6 @@ discourse_openid_connect:
     textarea: true
   openid_connect_match_by_email:
     default: true
+  openid_connect_use_pkce:
+    default: true
```

Setting this true by default could break existing installations which are incompatible with PKCE. We should make it false by default.

```suggestion
    default: false
```

Perhaps in future we could explore making it 'default true for new installations only'... but that can be a separate PR